Foundation.xcodeproj/project.pbxproj

```diff
@@ -1339,6 +1344,24 @@
 			path = Foundation;
 			sourceTree = "<group>";
 		};
+		617F57F01F1F5C4B0004F3F0 /* libcurlhelpers */ = {
```

Comment:
May be we should call the directory simply `libcurl` (because it contains libcurl abstractions) instead of `libcurlhelpers`